set-balance.md

```diff
@@ -81,15 +81,112 @@ A `Result` is the return value of an execution step.
+
+Retrieves the total balance of an account.  This includes both the free and
+reserved balances.
+
+```k
+    syntax Int ::= "total_balance" "(" AccountId ")" [function, functional]
+ // -----------------------------------------------------------------------
+    rule [[ total_balance(WHO) => FREE_BALANCE +Int RESERVED_BALANCE ]]
+         <account>
+           <accountID> WHO </accountID>
+           <freeBalance> FREE_BALANCE </freeBalance>
+           <reservedBalance> RESERVED_BALANCE </reservedBalance>
+           ...
+         </account>
+```
+
+### `can_slash`
+
+Determines if an account’s free balance is over the value provided.  This is
+often used to determine if an account has enough balance to cover a potential
+slash, hence the name.
+
+```k
+    syntax Bool ::= "can_slash" "(" AccountId "," Int ")" [function, functional]
+ // ----------------------------------------------------------------------------
+    rule [[ can_slash(WHO, AMOUNT) => FREE_BALANCE >=Int AMOUNT ]]
+         <account>
+           <accountID> WHO </accountID>
+           <freeBalance> FREE_BALANCE </freeBalance>
+           ...
+         </account>
+```
+
+### `total_issuance`
+
+Retrieves the total outstanding amount of currency outstanding.  This will
+always be equal to the sum of all free and reserved balances in all active
+accounts, except when the balances module is executing.
+
+```k
+    syntax Int ::= "total_issuance" [function, functional]
+ // ------------------------------------------------------
+    rule [[ total_issuance => TOTAL_ISSUANCE ]]
+         <totalIssuance> TOTAL_ISSUANCE </totalIssuance>
+```
+
+### `minimum_balance`
+
+Retrieves the minimum balance, also known as the existential deposit.
+
+Other than when this module is executing, no account will ever have a balance
+strictly between this and zero.
+
+```k
+    syntax Int ::= "minimum_balance" [function, functional]
+ // -------------------------------------------------------
+    rule [[ minimum_balance => EXISTENTIAL_DEPOSIT ]]
+         <totalIssuance> EXISTENTIAL_DEPOSIT </totalIssuance>
+```
+
+### `issue`
+
+Issues currency, creating an imbalance.
+
+This is not specified, since these semantics do not include the concept of an
+imbalance.  Without the concept of destructors and move semantics, it would be
+almost impossible to use correctly.
+
+### `burn`
+
+Burns currency.
+
+This is not part of the semantics for the same reason `burn` is not.
+
+### `free_balance`
+
+Gets the free balance of an account.
+
+Other than when this module is executing, this will never be strictly between
+`EXISTENTIAL_DEPOSIT` and zero.
+
+```k
+    syntax Int ::= "free_balance" "(" AccountId ")" [function, functional]
+ // ----------------------------------------------------------------------
+    rule [[ free_balance(WHO) => FREE_BALANCE ]]
+         <account>
+           <accountID> WHO </accountID>
+           <freeBalance> FREE_BALANCE </freeBalance>
+           ...
+         </account>
```

Comment:
let's group this with `total_balance` above since they are conceptually similar.